optifit backend/pushup_counter.py

```
@@ -0,0 +1,353 @@
+    last_rep_time = None
+
+    frame_count = 0
+    angle_history = collections.deque(maxlen=5)  # smoothing window
+    alignment_history = collections.deque(maxlen=5)
+
+    # Push-up specific thresholds
+    UP_THRESHOLD = 150          # angle considered up position
+    START_DOWN_THRESHOLD = 130  # start of descent detection
+    DOWN_THRESHOLD = 90         # angle considered down position (good depth)
+    ALIGNMENT_THRESHOLD = 70    # minimum body alignment score
+    HAND_POSITION_MARGIN = 0.05 # margin for hand position relative to shoulders
```

Ruff (0.14.1)

Local variable `HAND_POSITION_MARGIN` is assigned to but never used

Remove assignment to unused variable `HAND_POSITION_MARGIN`

(F841)